tests/conftest.py

```diff
@@ -741,9 +742,22 @@ async def create_client(
         auth_ok = await websocket.receive_json()
         assert auth_ok["type"] == TYPE_AUTH_OK
 
+        def _get_next_id() -> Generator[int, None, None]:
+            i = 0
+            while True:
+                yield (i := i + 1)
```

Comment:
Note: it won't increment if some calls use `send_json` and some calls use `send_json_auto_id` in the same test.
If that may cause an issue, then we would adjust the seed to not start at 1...

Reply:
The tests that are calling a mix will end up failing and test author can adjust.